### DISCUSSION TITLE
Which devices would you recommend? Door-/Window-sensors, thermostats, in-wall-switches, sensors for motion, brightness, air quality

### DISCUSSION THREAD

**Discussion by Ursusprimus** [Koenkk/zigbee2mqtt#6626] [Devices]
Hello,
I'm waiting for my the delvery of a zig-a-zig-ah! (zzh!)-adapter and plan to get a bunch of new zigbee-devices.
The list of compatible devices on https://www.zigbee2mqtt.io/ is quite extensive (and not all of them are available any more). But it doesn't tell which of them are good/bad.
So I'd be interested in your experiences of zigbee-devices, which could you recommend?
Relevant criteria to me are looks (especially window sensors should be small), quality and price (especially since a plan to buy several...).
I'm mostly interested in these device-types:
- Door-/Window-sensors, 
- thermostats, 
- in-wall-switches, 
- sensors for motion, brightness, air quality
- sensors for plants garden (ground humidity to check for watering, ...)
If you have other interesting ideas (I'm still quite new to smarthome), I'd be interested to hear about that, too.

**Comment by andylynch** [comment]
I have a zzh adapter too, .  I have the [Sonoff SNZB-04](https://sonoff.tech/product/smart-home-security/snzb-04) door/ window sensor which are small, unobtrusive, and easy to find on ebay or Amazon; am getting a couple more as I'm very happy with them.
Also have a [Sonoff SNZB-02](https://sonoff.tech/product/smart-home-security/snzb-02) indoor humidity / temperature sensor 

For motion sensors I have the IKEA ones [E1745](https://www.ikea.com/gb/en/p/tradfri-wireless-motion-sensor-white-70429913/) inside  and out - also inexpensive, well made, and  is IP44 rated so good at the front door / in the garden.  

IKEA Tradfri switches are also great.

I don't have a Zigbee thermostat (have  Tado which is also excellent, and integrates with my Home Assistant setup), but  the Tuya and Hive ones I would look at depended in where you are and what heating system you have.

**Reply by Ursusprimus** [reply]
Thanks for the tips, Andy.
The Amazon review on the SNZB-04 sometimes criticized short range and instable connectivity. You don't have these problems?

**Reply by timsterzel** [reply]
I have those problems with some sonoff devices. I have 4 sonoff tempertur and humidity sensors and two of them constantly loose connectivity.

**Comment by andylynch** [comment]
No, haven’t seen any problems like that.  Probably helps that it’s close to lights  which act as  routers in my Zigbee mesh.

**Comment by DubhAd** [comment]
> * Door-/Window-sensors,

I have a load of the [Xiaomi Aqara sensors](https://www.zigbee2mqtt.io/devices/MCCGQ11LM.html). They're pretty compact, don't stand out, and work well enough. They are only Zigbee 1.2, but the Konke devices are very fussy about the channels they work on (only three), and don't look as nice.

> * sensors for motion, brightness, air quality

The new [Xiaomi Mijia](https://www.zigbee2mqtt.io/devices/GZCGQ01LM.html) light sensors are really nice.

The [Aqara motion sensors](https://www.zigbee2mqtt.io/devices/RTCGQ11LM.html) are ok, the [Hue ones](https://www.zigbee2mqtt.io/devices/9290012607.html) are apparently better, but they cost a lot more. I also have the [Aqara temperature/humidity sensors](https://www.zigbee2mqtt.io/devices/WSDCGQ11LM.html) and they too work well.

**Reply by Ursusprimus** [reply]
Thanks for sharing; What are you using the light sensors for? 
I have a sensor (by innogy) that combines light and motion, which is useful for switching lights on when it's dark

**Reply by DubhAd** [reply]
For lighting control

The Xiaomi motion sensors also report light levels (though they're not terribly good light sensors), but only when there's motion, making them not that useful. I have these Mijia sensors, and some Z-Wave sensors, that I use for lighting control in various rooms.

**Comment by sjorge** [comment]
I’m using Ubisys S1 and S2 for my switches, they sit behind my original wall switches. I can turn on the power to the bulbs via z2m. And use the actual switch to control the lights themselves.

I’m seconding the sonoff contact sensors mentioned above, they work great and are cheap. There temp/humidity sensors also work well. I have about 7 contact and 6 temp/humidity, zero issues.

**Reply by Ursusprimus** [reply]
Just checked; the Ubisys switches are about 100 Euro, quite a lot. Yesterday I ordered a Yugusmart Zigbee-switch for 16 Euro on Amazon. Will take about one month. If it's any good, I might buy more and report here.
After the recommendations above I ordered 3 sonoff window switches (all in all just 21 Euro) and 1 Xiaomi light sensor (still on the way). I can't use/test some of these things yet because I'm still waiting for the zzh-hub (it's been delivered from England a week ago but seems to be stuck in customs now for a while, thanks to Brexit ...)
